src/main/java/com/indeed/jiraactions/api/ApiCaller.java

```diff
@@ -49,6 +50,7 @@ public JsonNode getJsonNode(final String url) throws IOException {
                 .build();
         final Response response = client.newCall(request).execute();
 
+        log.debug(""+cookieJar.loadForRequest(HttpUrl.parse(url)));
```

Comment:
Did you mean to leave this in?

Reply:
What does it even mean to call `toString()` on this List? Presumably you'd want some sort of `String.join` or something instead.

Also, prefer explicit `.toString()` rather than getting it by default by concatenating with a string.

Reply:
It was just a quick debug to make sure it was working, didn't mean to leave it in.